Pipfile

```diff
@@ -23,4 +23,7 @@ pydantic = "*"
 fastapi = ">=0.72.0"
 uvicorn = "*"
 boto3 = "*"
+requests = "*"
```

Comment:
Can we also move click to required packages here and in setup.cfg? This was a mistake on my part. 🤦‍♀️